test/server/routing.test.js

```diff
@@ -26,7 +26,7 @@ describe('Handling custom static routes', () => {
       },
       {
         path: '/static-route/:custom',
-        component: props => <p>Param: {props.params.custom}</p>
+        component: props => <p>Param: {props.requestData.params.custom}</p>
```

Comment:
Should this be `props._tapestry.requestData.params.custom`?

Reply:
Ah right, we're spreading it. I think we should keep the key as `_tapestry` to avoid any conflicts within the WP data

Reply:
No, this should stay the same as we're spreading the `_tapestry` object, so on a component level, you'll still be able to access all props from `appData` and `_tapestry` at the top level.

Reply:
Ok, I'm fine with changing that.